testing/ncids-css-testing/stories/components/nci-header/nci-header-logo.js

```diff
@@ -1,13 +1,31 @@
-import nciImg from './img/nci-img-logo-en.svg';
-import nciSvg from './img/nci-svg-logo.svg';
-import cbiittSvg from './img/cbiit-svg-logo.svg';
-import dccpsSVG from './img/dccps-img-logo.svg';
+import nciDesktop from './img/logo_NCI.svg';
+import nciMobile from './img/logo_NCI_mobile.svg';
+
+import nciEsDesktop from './img/logo_NCI_spanish.svg';
+import nciEsMobile from './img/logo_NCI_spanish_mobile.svg';
+
+import cbiitDesktop from './img/logo_CBIIT.svg';
+import cbiitMobile from './img/logo_CBIIT_mobile.svg';
+
+import dccpsDesktop from './img/logo_DCCPS.svg';
+import dccpsMobile from './img/logo_DCCPS_mobile.svg';
+
+import dcegDesktop from './img/logo_DCEG.svg';
+import dcegMobile from './img/logo_DCEG_mobile.svg';
+
+import mypartDesktop from './img/logo_MyPART.svg';
+import mypartMobile from './img/logo_MyPart_mobile.svg';
 
 // language=HTML
 export const nciImgLogo = `
```

Comment:
Can we please just make one function for the HTML and then pass in the 2 images to it?